third_party/CMakeLists.txt

```
@@ -38,4 +38,7 @@ elseif(OT_PLATFORM STREQUAL "qpg6095")
     add_subdirectory(Qorvo)
 elseif(OT_PLATFORM STREQUAL "samr21")
     add_subdirectory(microchip)
+elseif(OT_PLATFORM MATCHES "nrf")
```

Also for other instances.
```suggestion
elseif(OT_PLATFORM MATCHES "^nrf")
```

---

will update the same others as well